aksel.nav.no/website/pages/templates/soknad-oppsummeringsside-real-examples/dagpenger.tsx

```diff
@@ -0,0 +1,661 @@
+      <Page.Block as="main" width="text" gutters>
+        <VStack gap="8">
+          <Bleed marginInline={{ lg: "24" }}>
+            <Stack
+              gap="6"
+              direction={{ sm: "row-reverse", lg: "row" }}
+              justify={{ sm: "space-between", lg: "start" }}
+              wrap={false}
+            >
+              <Show above="sm">
+                <ApplicationPictogram />
+              </Show>
+              <Heading level="1" size="large">
+                Søknad om dagpenger
+              </Heading>
+            </Stack>
+          </Bleed>
+
+          <div data-aksel-template="form-summarypage-v1">
+            <Link href="#">
+              <ArrowLeftIcon aria-hidden /> Forrige steg
+            </Link>
+            <Box paddingBlock="6 5">
+              <Heading level="2" size="large">
+                Oppsummering
+              </Heading>
+            </Box>
+            <FormProgress activeStep={12} totalSteps={12}>
+              <FormProgress.Step href="#">Steg 1</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 2</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 3</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 4</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 5</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 6</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 7</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 8</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 9</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 10</FormProgress.Step>
+              <FormProgress.Step href="#">Steg 11</FormProgress.Step>
+              <FormProgress.Step href="#">Oppsummering</FormProgress.Step>
+            </FormProgress>
+          </div>
+
+          <GuidePanel poster>
+            <BodyLong spacing>
+              Nå kan du se over at alt er riktig før du sender inn søknaden. Ved
+              behov kan du endre opplysningene.
+            </BodyLong>
+            <BodyLong>
+              Når du har sendt inn søknaden kommer du til en kvitteringsside med
+              informasjon om veien videre. Der kan du også ettersende
+              dokumentasjon som mangler.
+            </BodyLong>
+          </GuidePanel>
+
+          <FormSummary>
+            <FormSummary.Header>
+              <FormSummary.Heading level="2">Personalia</FormSummary.Heading>
+              <FormSummary.EditLink href="#" />
+            </FormSummary.Header>
+            <FormSummary.Answers>
+              <FormSummary.Answer>
+                <FormSummary.Label>Navn</FormSummary.Label>
+                <FormSummary.Value>VOKSENDE TJA</FormSummary.Value>
+              </FormSummary.Answer>
+              <FormSummary.Answer>
+                <FormSummary.Label>Fødselsnummer</FormSummary.Label>
+                <FormSummary.Value>12345678912</FormSummary.Value>
```

Comment:
```suggestion
                <FormSummary.Value>123456 78912</FormSummary.Value>
```
From https://aksel.nav.no/god-praksis/artikler/skriveregler-i-nav
<img width="330" alt="Screenshot 2024-05-31 at 12 01 37" src="https://github.com/navikt/aksel/assets/26967723/cabe5db3-7164-4a95-adf1-1b088b7bfc5d">